src/components/PhoneNumberField/PhoneNumberField.js

```diff
@@ -1,4 +1,5 @@
 import { AsYouType, getCountryCallingCode, getCountries } from 'libphonenumber-js/custom';
+import { parsePhoneNumberFromString } from 'libphonenumber-js';
```

Comment:
@rahulpsingh will this import result in our bundle wrongly including all of the metadata for all countries? I tried to import on the previous line (from `'libphonenumber-js/custom'`) but it didn't work.

Reply:
ugh, it seems that it does indeed increase our built libs by a lot ☹️ I'll try and see how we can get it to work with our own custom metadata instead. Any suggestions would be appreciated.

Reply:
It should't include as we can check the implementation of it here: https://gitlab.com/catamphetamine/libphonenumber-js/-/blob/master/source/parsePhoneNumberFromString.js#L4

Reply:
Seems like it's coming from here: https://gitlab.com/catamphetamine/libphonenumber-js/-/blob/master/source/parsePhoneNumberFromString_.js#L3

Reply:
**Update**: I've just pushed a fix that uses our own custom metadata so that the build size is back to normal again :)